hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/ddl/HMSDDLExecutor.java

```diff
@@ -140,6 +142,10 @@ public void createTable(String tableName, MessageType storageSchema, String inpu
         newTb.setTableType(TableType.EXTERNAL_TABLE.toString());
       }
 
+      if (!tableProperties.containsKey(hive_metastoreConstants.META_TABLE_STORAGE)) {
+        tableProperties.put(hive_metastoreConstants.META_TABLE_STORAGE, DefaultStorageHandler.class.getName());
+      }
+
```

Comment:
The related document from Hive about the storage handlers: https://cwiki.apache.org/confluence/display/Hive/StorageHandlers

Reply:
It seems that the storage handler options conflict with row format and storage format options, does Hudi need those options though, need to confirm the affects.

Reply:
The row format and storage format are both working for tables that does not define any inputformat, Hudi table ways has a inputformat definition, so we are good here.

Reply:
thanks for the check